src/components/cargo.js

```diff
@@ -26,7 +26,7 @@ function Cargo(el, context) {
     const { cargo } = e.detail;
     const token = tokens[cargo.token] ? tokens[cargo.token].name : '-';
 
-    cargoSlot.src = `../../assets/${token}.png`;
+    cargoSlot.src = `assets/${token}.png`;
```

Comment:
Nitpick: this is an relative path, remember absolute paths are either `/..../.../` or the with the protocol `http://....`.